clang-tools-extra/docs/clang-tidy/checks/google/runtime-float.rst

```diff
@@ -0,0 +1,10 @@
+.. title:: clang-tidy - google-runtime-float
+
+google-runtime-float
+====================
+
+Finds uses of ``long double`` and suggests replacing them with 64-bit
+or 128-bit floating-point types.
```

Comment:
What are 128-bit floating point types?  I do not see any mention in the google guidelines.